platform-sdk/docs/base/test-support/test-support.md

```diff
@@ -61,4 +61,39 @@ executed in isolation (see `org.junit.jupiter.api.parallel.Isolated`).
 ### Context support
 
 The `@WithContext`, `@WithGlobalContext`, `@WithThreadLocalContext` annotation are documented in the chapter of 
-the [Context API](./../context/context.md).
\ No newline at end of file
+the [Context API](./../context/context.md).
+
+### Executor support
+
+The `@WithTestExecutor` annotation is used to inject a `TestExecutor` instance into a test. It can be applied at the
```

Comment:
```suggestion
The `@WithTestExecutor` annotation is used to inject a `TestExecutor` instance into a test.
```

Reply:
Oh, no I got it. you did a copy&paste error and the second sentence is splitted up.